File: python/code/134_pattern26.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(n):
        if(j>(n//2-i) and j<(n//2+i)):
            print(" ", end='')
        else: 
            print("*", end='')
    print()
    
# Output :-
# Enter n :- 5
# *****
# ** **
# *   *

File: c/code/134_pattern_26.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=0;j<n;j++) {
            if(j>(n/2-i)&&j<(n/2+i))
                printf(" ");
            else 
                printf("*");
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
*****
** **
*   *

*/